deploy/001_init.ts

```diff
@@ -24,7 +24,7 @@ const func: DeployFunction = async function ({
     contract: "Timelock",
     args: [
       deployer,
```

Comment:
Is there another script to change the admin from the deployer to the timelock after this is ran?